cassiopeia/preprocess/doublet_utils.pyx

```diff
@@ -173,7 +167,7 @@ def compute_lg_membership(
 
 
 def filter_inter_doublets(
-    at: pd.DataFrame, rule: float = 0.35, verbose: bool = False
+    at: pd.DataFrame, rule: float = 0.35
```

Comment:
Same as above - since we're updating logging functionality, can we log the parameter passed to this function as well?

Reply:
Same here
```python
if inter_doublet_threshold:
    logger.info(
        f"Filtering out inter-lineage group doublets with proportion {inter_doublet_threshold}..."
    )
    allele_table = d_utils.filter_inter_doublets(
        allele_table, rule=inter_doublet_threshold
    )
```
```